src/smc_sagews/smc_sagews/sage_salvus.py

```diff
@@ -3222,6 +3222,33 @@ def attach(*args):
         sys.stderr.flush()
 
 
+# Monkey-patch the save command
+def save(obj, filename=None, compress=True, **kwds):
+    """
+    Save ``obj`` to the file with name ``filename``.
+
+    .. NOTE::
+
+        sage_salvus.save() is a workaround to allow saving a function or class
+        loaded or defined at top level
+
+    .. WARNING::
+
+        Not applicable if pickle.dumps() is called instead of save()
+
+    .. SEEALSO::
+
+        :func:`sage_object.save`
```

Comment:
Does this work?  I bet it doesn't.     In the worksheet `sage_object.save` isn't even defined.

A serious problem with this docstring is that if 99.99% of people actually just types "save?" to learn how to use save, instead of immediately seeing useful information like they do now, e.g.,

> Save "obj" to the file with name "filename", which will have an
".sobj" extension added if it doesn't have one and if "obj" doesn't
have its own "save()" method, like e.g. Python tuples.  [...] For image objects and the like (which have their own "save()" method), you may have to specify a specific extension, e.g. ".png",
if you don't want the object to be saved as a Sage object (or
likewise, if "filename" could be interpreted as already having some
extension).

they see some useless discussion about saving functions that they could care less about.

A fix is to make the docstring of save just be the original one from sage with your appended to it!
You can do that programmatically (so no copy/paste) -- look for `__doc__` in the same file as save for examples.